integration/test/basic/basic_test.go

```diff
@@ -85,3 +149,23 @@ func waitForReleaseStatus(gsHelmClient *helmclient.Client, release string, statu
 	b := framework.NewExponentialBackoff(framework.ShortMaxWait, framework.LongMaxInterval)
 	return backoff.RetryNotify(operation, b, notify)
 }
+
+func waitForReleaseVersion(gsHelmClient *helmclient.Client, release string, version string) error {
+	operation := func() error {
+		rh, err := gsHelmClient.GetReleaseHistory(release)
+		if err != nil {
+			return microerror.Maskf(err, "could not retrieve release history")
+		}
+		if rh.Version != version {
+			return microerror.Newf("waiting for %q, current %q", version, rh.Version)
```

Comment:
Best practise so far is to create an error instance and use it. This here is inconsistent with the best practise and makes it harder to match against the specific error in case this becomes necessary.